vassal-doc/src/main/readme-referencemanual/ReferenceManual/GameModule.adoc

```diff
@@ -116,6 +116,9 @@ Simply type a name for each side and refer to that name in the restricted compon
 Only one player may be assigned to a side.
 When joining a game, players will be prompted to take one of the remaining available sides.
 Any number of observers (players who belong to no side) are allowed.
+
+When prompting for a side, a next default entry is offered. Certain names are skipped in this process; these are "Solo" and names beginning "Solitaire", "Solo:", "Moderator", "Referee" or name found in the <<GlobalTranslatableMessages.adoc#top, Global Translatable Messages>> property _VassalSkipSideList_. If used, the later should be defined as a comma-separated list of Sides.
+
```

Comment:
Why a Global Translatable message?

Reply:
It was my solution to the question of what happens if Translation(s) are added to the Definition of Sides component.
As the list of sides is translated, VassalSkipSideList will need to be translated also... or is there a better way ?

Reply:
> Why a Global Translatable message?

Now that Ive learnt a bit more about side translation in PlayerRoster, I want to take another go at this.

Reply:
To allow the PR to proceed, I will withdraw the extendable (VassalSkipSideList) element from this change and leave that to the wider design of side control features.

I'll also tackle the issue of side names that remain untranslated in a translated module.